examples/openmp/par_example.cpp

```diff
@@ -19,18 +19,15 @@
 #include "edm/spacepoint.hpp"
 #include "geometry/pixel_segmentation.hpp"
 #include "omp.h"
+#include "reader.hpp"
+
+using namespace std::literals::string_literals;
```

Comment:
Why do we need this? I don't seem to be able to spot any use of these literals?

Reply:
Right, became irrelevant.